lib/temporal/metadata.rb

```diff
@@ -6,37 +6,11 @@
 
 module Temporal
   module Metadata
-    ACTIVITY_TYPE = :activity
-    WORKFLOW_TASK_TYPE = :workflow_task
-    WORKFLOW_TYPE = :workflow
 
     class << self
       include Concerns::Payloads
 
-      def generate(type, data, namespace = nil)
```

Comment:
I removed this layer of indirection because we need to pass in different arguments depending on the type of metadata.